content/applications/finance/accounting/fiscal_localizations/localizations/india.rst

```diff
@@ -0,0 +1,170 @@
+=====
+India
+=====
+
+Introduction
+============
+
+The Indian localization has been improved and extended in Odoo v15. In this version, the next modules are available:
+
+- **l10n_in:** Adds accounting features for the Indian localization, which represent the minimal
+  configuration required for a company to operate in India. The main elements included in this module are two formats of Chart of Accounts i.e Indian Chart Of Accounts - Standard and Indian Chart Of Accounts - Schedule VI.
+
+- **l10n_in_edi:** Includes all technical and functional requirements to submit invoicing
+  through API to the government.
+
+Configuration
+=============
+
+Install the Indian localization modules
+----------------------------------------
+
+For this, go to *Apps* and search for ``l10n_in``. Then click *Install* in the module **Indian - E-invoicing**. This module has a dependency with **Indian - Accounting**. In case this last
+one is not installed, Odoo installs it automatically with E-invoicing.
+
+.. image:: media/india_edi_01.png
+   :align: center
+   :alt: Installation of the Indian localization module in Odoo Apps
+
+.. note::
+   When you install a database from scratch selecting India as country, Odoo will automatically install the base module: Indian - Accounting.
+
+Setup E-invoice Service
+------------------------
+
+Once the modules are installed, to be able to start the service,
+it’s necessary to set the username and password for the company.
+
+Go to :menuselection:`Accounting --> Configuration --> Settings` and
+look for the *Indian Electronic Invoicing* section.
+
+.. image:: media/india_edi_02.png
+   :align: center
+
+.. note::
+   For the creation of an API username and password please refer to this document:
+   https://service.odoo.co.in/einvoice_create_api_user
+
+Journal Configuration
+----------------------
+
+To configure you journals go to :menuselection:`Accounting --> Configuration --> Journals`:
+
+Open your sales journal. Under the Advanced Settings tab, in the *Electronic invoicing* field, check E-Invoice (IN) and Save. By default, it is *Checked* for sales journals.
```

Comment:
could you use `:guilabel:` for elements that are on-screen and `:menuselection:` for tabs?

```suggestion
Open your sales journal. Under the :menuselection:`Advanced Settings` tab, in the
:guilabel:`Electronic Invoicing` field, check :guilabel:`E-Invoice (IN)` and Save.```